config/software/google-protobuf.rb

```diff
@@ -27,6 +27,10 @@
 
 source git: "https://github.com/google/protobuf.git"
 
+# versions_list: https://github.com/protocolbuffers/protobuf/tags filter=*.tar.gz
+version("v3.17.0")    { source sha256: "eaba1dd133ac5167e8b08bc3268b2d33c6e9f2dcb14ec0f97f3d3eed9b395863" }
```

Comment:
So, the source here is the actual `git` repository and not a tarball, so I'm not sure we actually need to explicitly specify versions with SHAs.

Reply:
Yeah, I'll update PR accordingly